desktop/core/src/desktop/models.py

```diff
@@ -95,56 +95,26 @@ def hue_version():
   return HUE_VERSION
 
 def hue_image_version():
-  global HUE_IMAGE_VERSION
+  global IMAGE_VERSION
 
-  if HUE_IMAGE_VERSION is None:
+  if IMAGE_VERSION is None:
     p = get_run_root('cloudera', 'cdh_version.properties')
     if os.path.exists(p):
       build_version = _version_from_properties(open(p))
       if build_version:
-        HUE_IMAGE_VERSION = '%s' % (build_version)
+        IMAGE_VERSION = '%s' % (build_version)
 
-    elif os.getenv('HUE_CONF_DIR') and os.path.exists(os.getenv('HUE_CONF_DIR')):
-      cdw_path = os.getenv('HUE_CONF_DIR') + '/zhue.ini'
-      if os.path.exists(cdw_path):
-        hue_img_version = _version_from_hue_conf(open(cdw_path))
-        if hue_img_version:
-          HUE_IMAGE_VERSION = '%s' % (hue_img_version)
-
-  return HUE_IMAGE_VERSION
-
-def selected_hue_vw():
-  global SELECTED_HUE_VW
+    else:
+      IMAGE_VERSION = HUE_IMAGE_VERSION.get()
 
-  if SELECTED_HUE_VW is None:
-    if os.getenv('HUE_CONF_DIR') and os.path.exists(os.getenv('HUE_CONF_DIR')):
-      cdw_path = os.getenv('HUE_CONF_DIR') + '/zhue.ini'
-      if os.path.exists(cdw_path):
-        selected_hue_vw_name = _vw_name_from_hue_conf(open(cdw_path))
-        if selected_hue_vw_name:
-          SELECTED_HUE_VW = '%s' % (selected_hue_vw_name)
+  return IMAGE_VERSION
 
-  return SELECTED_HUE_VW
+def name_of_hue_host():
```

Comment:
Nit: Why not just `hue_host_name()`?

Reply:
The name_of_hue_host() function is called in get_config(). I cannot assign hue_host_name = hue_host_name(). So I changed the function name to name_of_hue_host().